velox/external/duckdb/CMakeLists.txt

```diff
@@ -13,4 +13,6 @@ add_subdirectory(tpch)
 
 add_library(duckdb duckdb.cpp parquet-amalgamation.cpp)
 
+add_compile_definitions(DISABLE_DUCKDB_REMOTE_INSTALL)
```

Comment:
I suspect this might need to be moved up, above the `add_library` that defines the source files.

Reply:
I don't think the order matters. I see that the generated make file has the definition locally.
```
/Users/deepak/workspace/ahana/velox/cmake-build-debugwiths3installed/velox/external/duckdb && /Library/Developer/CommandLineTools/usr/bin/c++ -DCODEGEN_ENABLED=1 -DDISABLE_DUCKDB_REMOTE_INSTALL -DVELOX_ENABLE_S3 -I/usr/local/opt/icu4c/include -I/Users/deepak/workspace/ahana/velox/. -isystem /usr/local/include -isystem /Users/deepak/workspace/ahana/velox/velox -isystem /Users/deepak/workspace/ahana/velox/velox/external -isystem /Users/deepak/workspace/ahana/velox/velox/external/duckdb -mavx2 -mfma -mavx -mf16c -mbmi2 -march=native -D USE_VELOX_COMMON_BASE -D HAS_UNCAUGHT_EXCEPTIONS -g -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX12.1.sdk -std=gnu++17 -o CMakeFiles/duckdb.dir/duckdb.cpp.o -c /Users/deepak/workspace/ahana/velox/velox/external/duckdb/duckdb.cpp
```
@mbasmanova can you try updating the buck scripts?